ui/pages/create-account/new-account.component.js

```diff
@@ -16,7 +17,19 @@ export default class NewAccountCreateForm extends Component {
 
   render() {
     const { newAccountName, defaultAccountName } = this.state;
-    const { history, createAccount, mostRecentOverviewPage } = this.props;
+    const {
+      history,
+      createAccount,
+      mostRecentOverviewPage,
+      accounts,
+    } = this.props;
+
+    const accountsNames = accounts.map((item) => item.name);
```

Comment:
Can this, and the `map` in `ui/components/app/modals/account-details-modal/account-details-modal.component.js`, be turned into a helper function?  Maybe `accountNameExists(proposedAccountName)`?  I'm always scared when we do similar operations in multiple spots.

Reply:
Fixed in c71199a60e2949308b7659f613fe02a48f46d79d.